source/manual/aws-iam-key-rotation.html.md

```diff
@@ -5,8 +5,8 @@ parent: "/manual.html"
 layout: manual_layout
 type: learn
 section: AWS
-last_reviewed_on: 2020-01-20
-review_in: 6 months
+last_reviewed_on: 2020-07-29
+review_in: 1 month
```

Comment:
I don't think this is necessary as the information in here won't be invalid if this 1 month time period passes. As the Trello card is to add extra information rather than contradicting information it should just enhance the documentation, should the trello card not be done someone might be rather confused as to why this is on such a short review cycle.